float64[] position_relative
float64 yaw_relative
float64 quality
